docker-assets/manageiq.conf

```diff
@@ -45,21 +30,21 @@ RewriteRule ^.*$ https://%{SERVER_NAME}%{REQUEST_URI} [L,R]
   ProxyPass /cws/ http://${MANAGEIQ_SERVICE_NAME}:9002/cws/
 
   ### API redirects
-  ProxyPass /api balancer://evmcluster_ws/api
-  ProxyPassReverse /api balancer://evmcluster_ws/api
+  ProxyPass /api https://${MANAGEIQ_SERVICE_NAME}
```

Comment:
I'm not sure what you mean ... we replaced `balancer://evmcluster_ws/api` with `https://${MANAGEIQ_SERVICE_NAME}`

It looks like the replacement should have been `https://${MANAGEIQ_SERVICE_NAME}/api`, right?